net-misc/netkit-talk/netkit-talk-0.17-r7.ebuild

```diff
@@ -0,0 +1,54 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit toolchain-funcs
+
+MY_P=netkit-ntalk-${PV}
+S="${WORKDIR}"/netkit-ntalk-${PV}
+
+DESCRIPTION="Netkit - talkd"
+HOMEPAGE="ftp://ftp.uk.linux.org/pub/linux/Networking/netkit/"
+SRC_URI="ftp://ftp.uk.linux.org/pub/linux/Networking/netkit/${MY_P}.tar.gz"
+
+LICENSE="BSD"
+SLOT="0"
+KEYWORDS="~alpha ~amd64 ~hppa ~ia64 ~mips ~ppc ~ppc64 ~sparc ~x86"
+IUSE="ipv6"
+
+DEPEND=">=sys-libs/ncurses-5.2:="
+BDEPEND="virtual/pkgconfig"
+RDEPEND="
+	${DEPEND}
+	virtual/inetd
+"
+
+PATCHES=( "${FILESDIR}"/${P}-time.patch )
+
+src_prepare() {
+	default
+	use ipv6 && eapply "${FILESDIR}"/${P}-ipv6.diff
```

Comment:
1. .diff -> .patch
2. Conditional patching is something we try to avoid, but I don't expect you to fix this in some abandoned codebase.
3. USE=ipv6 has dubious value in most cases (just something to know).

Reply:
Do you think it's worth dropping the USE flag and unconditionally applying the ipv6 fix?  To be honest I didn't look at the contents of the patch, just made sure that it built.

Reply:
It really depends on if how the patch works. But I wouldn't mess with it in this case given the fragility of the netkit stuff.

(That's fine by the way, part of why these fixes are great is because it gives me an opportunity to soapbox about random stuff to ensure you know it!)

Reply:
OK, checked the patch. Too janky for me. In theory, it'd be fine to unconditionally apply it, but please don't -- it's a bit scary with the bodged indentation and the rest of it.